Library/Homebrew/dependable.rb

```diff
@@ -23,7 +23,15 @@ def required?
     !build? && !optional? && !recommended?
   end
 
+  def build_and_run?
+    !build? && !run?
+  end
```

Comment:
This method doesn't seem to be used anywhere except in test case. Should we keep it?